lib/membrane/element/action.ex

```diff
@@ -201,6 +199,16 @@ defmodule Membrane.Element.Action do
   """
   @type end_of_stream_t :: {:end_of_stream, Pad.ref_t()}
 
+  @typedoc """
+  Terminates element with given reason.
+
+  Termination reason follows the OTP semantics:
+  - use `:normal` for graceful termination. Allowed only when termination was already requested
+    by parent, i.e. after `c:Membrane.Element.Base.handle_terminate_request/2` is called
+  - if reason is neither `:normal`, `:shutdown` nor `{:shutdown, term}`, an error is logged
```

Comment:
Do a Cmd+c, Cmd+v from previous suggestion

Reply:
Suggestion by courtesy of Grammarly:
```suggestion
  - use `:normal` for graceful termination. Allowed only when the parent already requested termination,
    i.e. after `c:Membrane.Element.Base.handle_terminate_request/2` is called
```